storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteIndexCache.java

```diff
@@ -573,8 +587,7 @@ public String toString() {
             return "Entry{" +
                     "offsetIndex=" + offsetIndex.file().getName() +
                     ", timeIndex=" + timeIndex.file().getName() +
-                    ", txnIndex=" + txnIndex.file().getName() +
-                    '}';
+                    ", txnIndex=" + txnIndexOpt.map(index -> index.file().getName()) + '}';
```

Comment:
I guess this will print "Optional.empty" here? Do we want to print empty string instead?